src/ssh/cockpitsshrelay.c

```diff
@@ -922,13 +921,18 @@ do_key_identity_auth (CockpitSshData *data)
   g_autofree gchar *pub_key_path = NULL;
   const gchar *msg;
 
+  gchar *default_identities[] = { "id_dsa", "id_ecdsa", "id_ecdsa_sk", "id_ed25519", "id_ed25519_sk", "id_rsa", NULL };
```

Comment:
Please add a comment here refering to the ssh manpage.

Reply:
This has been addressed as part of #14551.